ios/Exponent/Versioned/Core/Api/EXCategoryAction.h

```diff
@@ -0,0 +1,24 @@
+#import <Foundation/Foundation.h>
```

Comment:
Maybe you could move EXNotifications and this class to a new folder `Notifications` under `Api` directory